src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -355,10 +359,14 @@ def load_detail(self) -> Dict[str, list]:
             # collect from local files and concat in the memory
             flow_runs, node_runs = [], []
             for line_run_record_file in sorted(self._run_infos_folder.iterdir()):
+                if line_run_record_file.suffix.lower() != ".jsonl":
```

Comment:
Add some comments for the filtering logic here?

Reply:
Added.